### ISSUE TITLE
Read phenopackets

### ISSUE THREAD

**Issue by balhoff** [monarch-initiative/owlsim-v3#28]
Input phenotype profiles for analysis from a phenopacket file.

**Comment by balhoff** [comment]
@cmungall you mentioned reading a phenopacket in place of existing TSV reading. But the only TSV I can find is `ReadMappingsUtil.readPairwiseMappingsFromTsv` which seems like it's used to read in a list of individual comparisons to run, rather than phenotype annotations. Is that right? Should a phenopacket be read in along with the ontologies?

**Comment by cmungall** [comment]
You're right, currently the only way to load the phenotype annotations is as OWL

**Comment by balhoff** [comment]
Actually I think I was wrong—just came across `OWLLoader.loadDataFromTsv`, not sure how I missed that before. I'll add `loadDataFromPhenoPacket`.

**Comment by balhoff** [comment]
Initial implementation here: https://github.com/balhoff/owlsim-v3/commits/phenopacket-reading

Need to create some test data.
